From d174dce75e834593960b8522105451334d590864 Mon Sep 17 00:00:00 2001
From: Sandeep Patel <96694484+sandptel@users.noreply.github.com>
Date: Sat, 31 Aug 2024 06:35:14 +0530
Subject: [PATCH 2/2] Create
 0001-nixos-fhs-compatible-usr-share-changed-to-run-curren.patch

---
 ...ible-usr-share-changed-to-run-curren.patch | 602 ++++++++++++++++++
 1 file changed, 602 insertions(+)
 create mode 100644 0001-nixos-fhs-compatible-usr-share-changed-to-run-curren.patch

diff --git a/0001-nixos-fhs-compatible-usr-share-changed-to-run-curren.patch b/0001-nixos-fhs-compatible-usr-share-changed-to-run-curren.patch
new file mode 100644
index 0000000..84a168d
--- /dev/null
+++ b/0001-nixos-fhs-compatible-usr-share-changed-to-run-curren.patch
@@ -0,0 +1,602 @@
+From 06e5c2234fcddaa3f9918238915bb7d726062dfe Mon Sep 17 00:00:00 2001
+From: Sandeep Patel <96694484+sandptel@users.noreply.github.com>
+Date: Sat, 31 Aug 2024 06:25:12 +0530
+Subject: [PATCH] nixos fhs compatible (/usr/share changed to
+ /run/current-system/sw/usr/share)
+
+---
+ debian/copyright                            |  2 +-
+ usr/share/regolith-look/ayu-dark/root       | 10 +++++-----
+ usr/share/regolith-look/ayu-dark/wm         |  2 +-
+ usr/share/regolith-look/ayu-mirage/root     | 10 +++++-----
+ usr/share/regolith-look/ayu-mirage/wm       |  2 +-
+ usr/share/regolith-look/ayu/root            | 10 +++++-----
+ usr/share/regolith-look/ayu/wm              |  2 +-
+ usr/share/regolith-look/blackhole/root      | 12 ++++++------
+ usr/share/regolith-look/blackhole/wm        |  2 +-
+ usr/share/regolith-look/dracula/root        | 14 +++++++-------
+ usr/share/regolith-look/dracula/wm          |  2 +-
+ usr/share/regolith-look/gruvbox/root        | 12 ++++++------
+ usr/share/regolith-look/gruvbox/wm          |  2 +-
+ usr/share/regolith-look/i3-default/root     | 10 +++++-----
+ usr/share/regolith-look/i3-default/wm       |  2 +-
+ usr/share/regolith-look/lascaille/root      | 12 ++++++------
+ usr/share/regolith-look/lascaille/wm        |  2 +-
+ usr/share/regolith-look/nevil/root          | 10 +++++-----
+ usr/share/regolith-look/nevil/wm            |  2 +-
+ usr/share/regolith-look/nord/root           | 14 +++++++-------
+ usr/share/regolith-look/nord/wm             |  2 +-
+ usr/share/regolith-look/solarized-dark/root | 12 ++++++------
+ usr/share/regolith-look/solarized-dark/wm   |  2 +-
+ 23 files changed, 75 insertions(+), 75 deletions(-)
+
+diff --git a/debian/copyright b/debian/copyright
+index b6246d4..d7777d3 100644
+--- a/debian/copyright
++++ b/debian/copyright
+@@ -64,4 +64,4 @@ License: GPL-3.0+
+  along with this program. If not, see <https://www.gnu.org/licenses/>.
+  .
+  On Debian systems, the complete text of the GNU General
+- Public License version 3 can be found in "/usr/share/common-licenses/GPL-3".
++ Public License version 3 can be found in "/run/current-system/sw/usr/share/common-licenses/GPL-3".
+diff --git a/usr/share/regolith-look/ayu-dark/root b/usr/share/regolith-look/ayu-dark/root
+index 51d9389..37cc251 100644
+--- a/usr/share/regolith-look/ayu-dark/root
++++ b/usr/share/regolith-look/ayu-dark/root
+@@ -58,7 +58,7 @@ gtk.icon_theme_name: Arc
+ ! -- Compositor Config
+ ! --------------------------------------------
+ 
+-regolith.compositor.picom.config: /usr/share/regolith-look/ayu-dark/picom.conf
++regolith.compositor.picom.config: /run/current-system/sw/usr/share/regolith-look/ayu-dark/picom.conf
+ ! -- Other compositor configs may be specified as desired:
+ ! -- regolith.compositor.compton.config
+ 
+@@ -102,13 +102,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/ayu-dark/i3xrocks"
+-#include "/usr/share/regolith-look/ayu-dark/wm"
+-#include "/usr/share/regolith-look/ayu-dark/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu-dark/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu-dark/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu-dark/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/ayu-dark/wm b/usr/share/regolith-look/ayu-dark/wm
+index 9bf3af7..00c10a7 100644
+--- a/usr/share/regolith-look/ayu-dark/wm
++++ b/usr/share/regolith-look/ayu-dark/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/ayu-dark/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/ayu-dark/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/ayu-mirage/root b/usr/share/regolith-look/ayu-mirage/root
+index 77aeb86..6df658a 100644
+--- a/usr/share/regolith-look/ayu-mirage/root
++++ b/usr/share/regolith-look/ayu-mirage/root
+@@ -58,7 +58,7 @@ gtk.icon_theme_name: Arc
+ ! -- Compositor Config
+ ! --------------------------------------------
+ 
+-regolith.compositor.picom.config: /usr/share/regolith-look/ayu-mirage/picom.conf
++regolith.compositor.picom.config: /run/current-system/sw/usr/share/regolith-look/ayu-mirage/picom.conf
+ ! -- Other compositor configs may be specified as desired:
+ ! -- regolith.compositor.compton.config
+ 
+@@ -102,13 +102,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/ayu-mirage/i3xrocks"
+-#include "/usr/share/regolith-look/ayu-mirage/wm"
+-#include "/usr/share/regolith-look/ayu-mirage/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu-mirage/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu-mirage/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu-mirage/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/ayu-mirage/wm b/usr/share/regolith-look/ayu-mirage/wm
+index 8195fee..32cd042 100644
+--- a/usr/share/regolith-look/ayu-mirage/wm
++++ b/usr/share/regolith-look/ayu-mirage/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/ayu-mirage/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/ayu-mirage/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/ayu/root b/usr/share/regolith-look/ayu/root
+index f8e2f19..ce4207a 100644
+--- a/usr/share/regolith-look/ayu/root
++++ b/usr/share/regolith-look/ayu/root
+@@ -58,7 +58,7 @@ gtk.icon_theme_name: Arc
+ ! -- Compositor Config
+ ! --------------------------------------------
+ 
+-regolith.compositor.picom.config: /usr/share/regolith-look/ayu/picom.conf
++regolith.compositor.picom.config: /run/current-system/sw/usr/share/regolith-look/ayu/picom.conf
+ ! -- Other compositor configs may be specified as desired:
+ ! -- regolith.compositor.compton.config
+ 
+@@ -101,13 +101,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/ayu/i3xrocks"
+-#include "/usr/share/regolith-look/ayu/wm"
+-#include "/usr/share/regolith-look/ayu/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/ayu/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/ayu/wm b/usr/share/regolith-look/ayu/wm
+index 41f8051..2ff4b7f 100644
+--- a/usr/share/regolith-look/ayu/wm
++++ b/usr/share/regolith-look/ayu/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/ayu/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/ayu/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/blackhole/root b/usr/share/regolith-look/blackhole/root
+index 8f849e8..c10e6fd 100644
+--- a/usr/share/regolith-look/blackhole/root
++++ b/usr/share/regolith-look/blackhole/root
+@@ -56,7 +56,7 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ 
+ ! -- Specify either a complete file path to an image
+ 
+-regolith.wallpaper.file: /usr/share/backgrounds/regolith-blackhole-v1.0.0.png
++regolith.wallpaper.file: /run/current-system/sw/usr/share/backgrounds/regolith-blackhole-v1.0.0.png
+ regolith.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -77,7 +77,7 @@ regolith.wallpaper.color.shading.type:
+ ! -- Specify either a complete file path to an image
+ ! -- and (optionally) options how to display the file
+ 
+-regolith.lockscreen.wallpaper.file: /usr/share/backgrounds/regolith-blackhole-v1.0.0.png
++regolith.lockscreen.wallpaper.file: /run/current-system/sw/usr/share/backgrounds/regolith-blackhole-v1.0.0.png
+ regolith.lockscreen.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -93,13 +93,13 @@ regolith.lockscreen.wallpaper.color.shading.type:
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/blackhole/i3xrocks"
+-#include "/usr/share/regolith-look/blackhole/wm"
+-#include "/usr/share/regolith-look/blackhole/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/blackhole/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/blackhole/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/blackhole/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/blackhole/wm b/usr/share/regolith-look/blackhole/wm
+index 7147d2c..8d4978c 100644
+--- a/usr/share/regolith-look/blackhole/wm
++++ b/usr/share/regolith-look/blackhole/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/blackhole/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/blackhole/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/dracula/root b/usr/share/regolith-look/dracula/root
+index e2f02d1..f1856d3 100644
+--- a/usr/share/regolith-look/dracula/root
++++ b/usr/share/regolith-look/dracula/root
+@@ -14,7 +14,7 @@ regolith.look: dracula
+ 
+ ! -- Specify either a complete file path to an image
+ 
+-regolith.wallpaper.file: /usr/share/regolith-look/dracula/dracula-regolith.png
++regolith.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/dracula/dracula-regolith.png
+ regolith.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -35,7 +35,7 @@ regolith.wallpaper.color.shading.type:
+ ! -- Specify either a complete file path to an image
+ ! -- and (optionally) options how to display the file
+ 
+-regolith.lockscreen.wallpaper.file: /usr/share/regolith-look/dracula/dracula-regolith.png
++regolith.lockscreen.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/dracula/dracula-regolith.png
+ regolith.lockscreen.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -58,7 +58,7 @@ gtk.icon_theme_name: Dracula
+ ! -- Compositor Config
+ ! --------------------------------------------
+ 
+-regolith.compositor.picom.config: /usr/share/regolith-look/dracula/picom.conf
++regolith.compositor.picom.config: /run/current-system/sw/usr/share/regolith-look/dracula/picom.conf
+ ! -- Other compositor configs may be specified as desired:
+ ! -- regolith.compositor.compton.config
+ 
+@@ -103,13 +103,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/dracula/i3xrocks"
+-#include "/usr/share/regolith-look/dracula/wm"
+-#include "/usr/share/regolith-look/dracula/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/dracula/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/dracula/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/dracula/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/dracula/wm b/usr/share/regolith-look/dracula/wm
+index babaa7e..a3da69b 100644
+--- a/usr/share/regolith-look/dracula/wm
++++ b/usr/share/regolith-look/dracula/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/dracula/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/dracula/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/gruvbox/root b/usr/share/regolith-look/gruvbox/root
+index ac14791..bc3a784 100644
+--- a/usr/share/regolith-look/gruvbox/root
++++ b/usr/share/regolith-look/gruvbox/root
+@@ -14,7 +14,7 @@ regolith.look: gruvbox
+ 
+ ! -- Specify either a complete file path to an image
+ 
+-regolith.wallpaper.file: /usr/share/regolith-look/gruvbox/gruvbox-dark-regolith2-noText-GMFranceschini.svg
++regolith.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/gruvbox/gruvbox-dark-regolith2-noText-GMFranceschini.svg
+ regolith.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -35,7 +35,7 @@ regolith.wallpaper.color.shading.type:
+ ! -- Specify either a complete file path to an image
+ ! -- and (optionally) options how to display the file
+ 
+-regolith.lockscreen.wallpaper.file: /usr/share/regolith-look/gruvbox/gruvbox-dark-regolith2-noText-GMFranceschini.svg
++regolith.lockscreen.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/gruvbox/gruvbox-dark-regolith2-noText-GMFranceschini.svg
+ regolith.lockscreen.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -99,13 +99,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/gruvbox/i3xrocks"
+-#include "/usr/share/regolith-look/gruvbox/wm"
+-#include "/usr/share/regolith-look/gruvbox/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/gruvbox/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/gruvbox/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/gruvbox/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/gruvbox/wm b/usr/share/regolith-look/gruvbox/wm
+index 770cc90..ae2d930 100644
+--- a/usr/share/regolith-look/gruvbox/wm
++++ b/usr/share/regolith-look/gruvbox/wm
+@@ -11,7 +11,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/gruvbox/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/gruvbox/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/i3-default/root b/usr/share/regolith-look/i3-default/root
+index dc0af6e..3728c84 100644
+--- a/usr/share/regolith-look/i3-default/root
++++ b/usr/share/regolith-look/i3-default/root
+@@ -58,7 +58,7 @@ gtk.icon_theme_name: Adwaita
+ ! -- Compositor Config
+ ! --------------------------------------------
+ 
+-regolith.compositor.picom.config: /usr/share/regolith-look/i3-default/picom.conf
++regolith.compositor.picom.config: /run/current-system/sw/usr/share/regolith-look/i3-default/picom.conf
+ ! -- Other compositor configs may be specified as desired:
+ ! -- regolith.compositor.compton.config
+ 
+@@ -151,13 +151,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/i3-default/i3xrocks"
+-#include "/usr/share/regolith-look/i3-default/wm"
+-#include "/usr/share/regolith-look/i3-default/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/i3-default/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/i3-default/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/i3-default/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/i3-default/wm b/usr/share/regolith-look/i3-default/wm
+index 68c99dd..b87e574 100644
+--- a/usr/share/regolith-look/i3-default/wm
++++ b/usr/share/regolith-look/i3-default/wm
+@@ -6,7 +6,7 @@
+ #define i3wm_gaps_outer_size             0
+ #define i3wm_bar_position                bottom
+ 
+-ilia.stylesheet: /usr/share/regolith-look/i3-default/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/i3-default/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/lascaille/root b/usr/share/regolith-look/lascaille/root
+index 48ab037..5e1d267 100644
+--- a/usr/share/regolith-look/lascaille/root
++++ b/usr/share/regolith-look/lascaille/root
+@@ -17,7 +17,7 @@ regolith.look: lascaille
+ ! -- Specify either a complete file path to an image
+ ! -- and (optionally) options how to display the file
+ 
+-regolith.wallpaper.file: /usr/share/regolith-look/lascaille/ESP_016895_1525_desktop.jpg
++regolith.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/lascaille/ESP_016895_1525_desktop.jpg
+ regolith.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -38,7 +38,7 @@ regolith.wallpaper.color.shading.type:
+ ! -- Specify either a complete file path to an image
+ ! -- and (optionally) options how to display the file
+ 
+-regolith.lockscreen.wallpaper.file: /usr/share/regolith-look/lascaille/pia21972.png
++regolith.lockscreen.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/lascaille/pia21972.png
+ regolith.lockscreen.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -104,9 +104,9 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/lascaille/i3xrocks"
+-#include "/usr/share/regolith-look/lascaille/wm"
+-#include "/usr/share/regolith-look/lascaille/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/lascaille/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/lascaille/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/lascaille/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+@@ -114,4 +114,4 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- NOTE: The following script path can be changed to support alternative look configurations
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/lascaille/wm b/usr/share/regolith-look/lascaille/wm
+index 2731e8b..ad7e630 100644
+--- a/usr/share/regolith-look/lascaille/wm
++++ b/usr/share/regolith-look/lascaille/wm
+@@ -7,7 +7,7 @@
+ #define i3wm_bar_position                bottom
+ 
+ 
+-ilia.stylesheet: /usr/share/regolith-look/lascaille/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/lascaille/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/nevil/root b/usr/share/regolith-look/nevil/root
+index b63fa17..274d251 100644
+--- a/usr/share/regolith-look/nevil/root
++++ b/usr/share/regolith-look/nevil/root
+@@ -98,7 +98,7 @@ regolith.lockscreen.wallpaper.color.shading.type:
+ ! -- Compositor Config
+ ! --------------------------------------------
+ 
+-regolith.compositor.picom.config: /usr/share/regolith-look/nevil/picom.conf
++regolith.compositor.picom.config: /run/current-system/sw/usr/share/regolith-look/nevil/picom.conf
+ ! -- Other compositor configs may be specified as desired:
+ ! -- regolith.compositor.compton.config
+ 
+@@ -106,13 +106,13 @@ regolith.compositor.picom.config: /usr/share/regolith-look/nevil/picom.conf
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/nevil/i3xrocks"
+-#include "/usr/share/regolith-look/nevil/wm"
+-#include "/usr/share/regolith-look/nevil/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/nevil/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/nevil/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/nevil/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/nevil/wm b/usr/share/regolith-look/nevil/wm
+index 10faef5..68c404a 100644
+--- a/usr/share/regolith-look/nevil/wm
++++ b/usr/share/regolith-look/nevil/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/nevil/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/nevil/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/nord/root b/usr/share/regolith-look/nord/root
+index 8806da7..4b2de17 100644
+--- a/usr/share/regolith-look/nord/root
++++ b/usr/share/regolith-look/nord/root
+@@ -14,7 +14,7 @@ regolith.look: nord
+ 
+ ! -- Specify either a complete file path to an image
+ 
+-regolith.wallpaper.file: /usr/share/regolith-look/nord/nord-regolith2-noText-GMFranceschini.svg
++regolith.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/nord/nord-regolith2-noText-GMFranceschini.svg
+ regolith.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -35,7 +35,7 @@ regolith.wallpaper.color.shading.type:
+ ! -- Specify either a complete file path to an image
+ ! -- and (optionally) options how to display the file
+ 
+-regolith.lockscreen.wallpaper.file: /usr/share/regolith-look/nord/nord-dark.png
++regolith.lockscreen.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/nord/nord-dark.png
+ regolith.lockscreen.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -58,7 +58,7 @@ gtk.icon_theme_name: Arc
+ ! -- Compositor Config
+ ! --------------------------------------------
+ 
+-regolith.compositor.picom.config: /usr/share/regolith-look/nord/picom.conf
++regolith.compositor.picom.config: /run/current-system/sw/usr/share/regolith-look/nord/picom.conf
+ ! -- Other compositor configs may be specified as desired:
+ ! -- regolith.compositor.compton.config
+ 
+@@ -101,13 +101,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/nord/i3xrocks"
+-#include "/usr/share/regolith-look/nord/wm"
+-#include "/usr/share/regolith-look/nord/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/nord/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/nord/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/nord/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/nord/wm b/usr/share/regolith-look/nord/wm
+index e13f78c..7e74911 100644
+--- a/usr/share/regolith-look/nord/wm
++++ b/usr/share/regolith-look/nord/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/nord/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/nord/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+diff --git a/usr/share/regolith-look/solarized-dark/root b/usr/share/regolith-look/solarized-dark/root
+index 78b1862..3f9a1b3 100644
+--- a/usr/share/regolith-look/solarized-dark/root
++++ b/usr/share/regolith-look/solarized-dark/root
+@@ -14,7 +14,7 @@ regolith.look: solarized-dark
+ 
+ ! -- Specify either a complete file path to an image
+ 
+-regolith.wallpaper.file: /usr/share/regolith-look/solarized-dark/cojdev_github_io-lowpoly.png
++regolith.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/solarized-dark/cojdev_github_io-lowpoly.png
+ regolith.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -35,7 +35,7 @@ regolith.wallpaper.color.shading.type:
+ ! -- Specify either a complete file path to an image
+ ! -- and (optionally) options how to display the file
+ 
+-regolith.lockscreen.wallpaper.file: /usr/share/regolith-look/solarized-dark/cojdev_github_io-lowpoly.png
++regolith.lockscreen.wallpaper.file: /run/current-system/sw/usr/share/regolith-look/solarized-dark/cojdev_github_io-lowpoly.png
+ regolith.lockscreen.wallpaper.options: zoom
+ 
+ !-- *Or* specify a color
+@@ -92,13 +92,13 @@ gtk.monospace_font_name: gtk_monospace_font_name
+ ! -- Component resources
+ ! --------------------------------------------
+ 
+-#include "/usr/share/regolith-look/solarized-dark/i3xrocks"
+-#include "/usr/share/regolith-look/solarized-dark/wm"
+-#include "/usr/share/regolith-look/solarized-dark/gnome-terminal"
++#include "/run/current-system/sw/usr/share/regolith-look/solarized-dark/i3xrocks"
++#include "/run/current-system/sw/usr/share/regolith-look/solarized-dark/wm"
++#include "/run/current-system/sw/usr/share/regolith-look/solarized-dark/gnome-terminal"
+ 
+ ! --------------------------------------------
+ ! -- Loader Path - Path to script responsible 
+ ! -- for configuring UI from Xres values
+ ! --------------------------------------------
+ 
+-regolith.look.loader: /usr/share/regolith-look/default_loader.sh
++regolith.look.loader: /run/current-system/sw/usr/share/regolith-look/default_loader.sh
+diff --git a/usr/share/regolith-look/solarized-dark/wm b/usr/share/regolith-look/solarized-dark/wm
+index 249ecdc..fc6f21e 100644
+--- a/usr/share/regolith-look/solarized-dark/wm
++++ b/usr/share/regolith-look/solarized-dark/wm
+@@ -10,7 +10,7 @@
+ #define QUOTE(x) Q(x)
+ #define glyph typeface_bar_glyph_workspace
+ 
+-ilia.stylesheet: /usr/share/regolith-look/solarized-dark/ilia.css
++ilia.stylesheet: /run/current-system/sw/usr/share/regolith-look/solarized-dark/ilia.css
+ 
+ wm.bar.font: gtk_monospace_font_name
+ wm.bar.position: i3wm_bar_position
+-- 
+2.44.1
+
-- 
2.44.1

